### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yashovardhan**
Update README according to new structure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,37 +37,44 @@ or
...
 ```
 
-Please refer to the native SDKs for platform-specific configuration.
+## 🌟 Configuration
+
+Add `web3auth_flutter` as a dependency to your `pubspec.yaml`
+
+```yaml
+dependencies:
+  web3auth_flutter:
```
Comment (chaitanyapotti) [inline README.md:46]: use 

```yaml
web3auth_flutter: ^1.1.0
```

File: README.md [hunk 2]
```diff
@@ -15,37 +37,44 @@ or
...
-        redirectUrl: Uri.parse('torusapp://org.torusresearch.flutter.web3authexample'),
-        whiteLabel: WhiteLabelData(
-            dark: true, name: "Web3Auth Flutter App", theme: themeMap)));
-```
+- Android `AndoidManifest.xml` (required)
+- iOS `Info.plist` (optional)
+
+## 🩹 Examples
+
+Checkout the examples for your preferred blockchain and platform in our [examples repository](https://github.com/Web3Auth/examples/)
```
Comment (chaitanyapotti) [inline README.md:68]: remove the examples

File: README.md [hunk 3]
```diff
@@ -15,37 +37,44 @@ or
...
+## 🌐 Demo
 
-// Login
-final Web3AuthResponse response = await Web3AuthFlutter.login(LoginParams(loginProvider: Provider.google));
+Checkout the [Web3Auth Demo](https://demo-app.web3auth.io/) to see how Web3Auth can be used in an application.
 
-// Logout
-await Web3AuthFlutter.logout();
-````
+Further checkout the [example folder](https://github.com/Web3Auth/web3auth-react-native-sdk/tree/master/example) within this repository, which contains a sample app.
```
Comment (chaitanyapotti) [inline README.md:74]: this must be the flutter url

File: README.md [hunk 4]
```diff
@@ -15,37 +37,44 @@ or
...
+
+## 🩹 Examples
+
+Checkout the examples for your preferred blockchain and platform in our [examples repository](https://github.com/Web3Auth/examples/)
+
+## 🌐 Demo
 
-// Login
-final Web3AuthResponse response = await Web3AuthFlutter.login(LoginParams(loginProvider: Provider.google));
+Checkout the [Web3Auth Demo](https://demo-app.web3auth.io/) to see how Web3Auth can be used in an application.
```
Comment (chaitanyapotti) [inline README.md:72]: use the demo for flutter

File: README.md (hunk 2 above)
Comment (chaitanyapotti) [reply README.md:68]: this is fine

File: README.md (hunk 1 above)
Comment (yashovardhan) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (yashovardhan) [reply README.md:72]: there's no hosted demo for flutter

Comment (shahbaz17) [reply README.md:72]: I think you can use something similar to Android readme.

> Further checkout the [example folder](https://github.com/Web3Auth/web3auth-flutter-sdk/tree/master/example) within this repository, which contains a sample app.

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**